src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/RejectApplications.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@model SFA.DAS.LevyTransferMatching.Web.Models.Pledges.RejectApplicationsViewModel
+
+@{
+    ViewData["Title"] = "Pledges";
```

Comment:
Could this be changed to "Reject Applications"?

Reply:
This is now updated to "Reject Applications".